dashboard/app/assets/javascripts/progress.js

```diff
@@ -101,5 +108,12 @@ window.dashboard.progress = (function () {
     })));
   };
 
+  progress.renderCourseProgress = function (scriptData) {
+    $('.user-stats-block').prepend(React.renderToStaticMarkup(React.createElement(dashboard.CourseProgress, {
+      stages: scriptData.stages
+    })));
+    progress.populateProgress(scriptData.name);
```

Comment:
The only issue is that right now the component shares `$('.user-stats-block')` with the user stats key.